pipeline/data/parallel_importer.py

```diff
@@ -156,6 +197,14 @@ def augment(output_prefix: str, aug_modifier: str, src: str, trg: str):
 
     corpus = read_corpus_tsv(compressed_src, compressed_trg, uncompressed_src, uncompressed_trg)
 
+    # Special modifier aug-short just samples and finishes
+    if aug_modifier == "aug-short":
+        # Sample params are hardcoded for now
+        modifier = SampleModifier(src_lang=src, trg_lang=trg, n=400, min_words=1, max_words=2)
```

Comment:
Since the segments are so short, 1-2 words, we should do deduplication. So in this case it would require sampling more just in case, maybe 5k, then dedupe and sample 1k.

Reply:
I don't know if it's useful here but we also have `datasets.sample_corpus`. It's a bit more focused on streaming to work with huge datasets like HPLT. Maybe it's worth doing the first pass using it in case we want to sample from bigger datasets in the future.